models/models.js

```diff
@@ -1,9 +1,18 @@
 const db = require('../db/connection');
-// const {topicData} = require (`../db/data/${ENV}-data/index`);
 
 exports.getTopicInfo = () =>{
-    return db.query("SELECT * FROM topics;")
+    return db.query('SELECT * FROM topics;')
     .then(({rows}) => {
         return (rows);
     });
-};
\ No newline at end of file
+};
+
+exports.getArticleInfo = () => {
+    const sqlquery = 'SELECT articles.author,articles.title,articles.article_id,articles.topic,articles.created_at,articles.votes,COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;'
+    return db.query(sqlquery)
+    .then(({rows}) => {
+        // console.log(rows)
```

Comment:
Please remove :)

Reply:
I removed it but not sure how it cropped up; cant find in mine either :(